src/app/fdctl/run/tiles/fd_poh.c

```diff
@@ -562,6 +565,14 @@ fd_ext_poh_initialize( ulong         hashcnt_duration_ns, /* See clock comments
      since it is used frequently. */
   ctx->hashcnt_per_slot = ticks_per_slot*hashcnt_per_tick;
 
+  if( FD_UNLIKELY( ctx->hashcnt_per_tick==1UL ) ) {
+    /* Low power producer, maximum of one microblock per tick in the slot */
+    ctx->max_microblocks_per_slot = ctx->ticks_per_slot;
+  } else {
+    ctx->max_microblocks_per_slot = 817UL; /* FD_PACK_MAX_MICROBLOCKS_PER_BLOCK */
```

Comment:
Could you move the constant to disco_base or something and use the actual constant?